finalPrep/README.md

```diff
@@ -18,37 +18,41 @@ Some things to consider as you refactor include...
 
 Take your time as you go back and refactor your exercises from each section. We've included a handy checklist for you to go through below.
 
-- [ ] I have refactored my `section1` exercises to the best of my ability
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
-- [ ] I have refactored my `section4` exercises to the best of my ability
+- [x] I have refactored my `section1` exercises to the best of my ability
+- [x] I have refactored my `section2` exercises to the best of my ability
+- [x] I have refactored my `section3` exercises to the best of my ability
+- [x] I have refactored my `section4` exercises to the best of my ability
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [x] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+`This will definitely change when I have a better understanding of standard weekly group work. I will be able to get more granular with homework v. project v. study time. And I will build in time to meet with collaborators based on everyone's availability.`
+
+- `Add Week 1 Screenshot Here`![](assets/README-826b900a.png)
+
```

Comment:
Great stuff here! Looks like you have a routine set up, including time planned for weekend work. Things inevitably change, so I am glad to see the Saturday morning reflections to adjust for the upcoming week.
- I try to take stock of what went well, and what didn't go so well to try to be realistic about how I allot time in the upcoming week.
- When I know what I need to work on, I will edit calendar events to bring more focus into the task at hand for that block. Sometimes, a 2 hour homework block would be unfocussed for me, so I would pick a thing, maybe 2, for that time.

Reply:
Thanks so much for your thorough feedback and support. Just reviewed in detail after a busy couple of  weeks at work. Duly noted all your suggestions and tips!